# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: Cargo.toml
# Commit: a651d4323066
# Full Hash: a651d43230664837c308663382ccc3bcd7298dc2
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r be008ddb2418 -r a651d4323066 Cargo.toml
--- a/Cargo.toml	Fri Jan 27 21:25:50 2023 +0000
+++ b/Cargo.toml	Fri Jan 27 21:25:51 2023 +0000
@@ -104,9 +104,6 @@
 # Patch getrandom 0.7 to 0.8
 rand = { path = "build/rust/rand" }
 
-# Patch env_logger 0.8 to 0.9
-env_logger = { path = "build/rust/env_logger" }
-
 # Patch parking_lot 0.12 down to 0.11, which is compatible for most crates that use it, to avoid
 # dependencies on windows-sys.
 parking_lot = { path = "build/rust/parking_lot" }